test/src/main/java/org/apache/accumulo/test/ExternalCompactionIT.java

```diff
@@ -193,22 +195,55 @@ public boolean accept(Key k, Value v) {
 
   }
 
+  @Before
+  public void setUp() throws Exception {
+    if (SharedMiniClusterBase.getCluster() == null) {
+      SharedMiniClusterBase.startMiniClusterWithConfig(this);
+    }
+  }
+
+  private void stopProcess(Process process) throws Exception {
```

Comment:
I saw this is called several times in some places with the same pattern. Could do the null check inside this method, and the arg list could be varargs: ```private void stopProcess(ProcessInfo ... processes) ... ```

Reply:
Yes, good idea.

Reply:
Addressed in da6017f